### ISSUE TITLE
Update Tutorials README with the FRB usecase

### ISSUE THREAD

**Issue by elboyran** [dianna-ai/dianna#719]
[README](https://github.com/dianna-ai/dianna/tree/main/tutorials#readme)

The dataset (we need some cute logo ;-) ) and the link to the tutorial.

**Comment by elboyran** [comment]
Best to do after #667 is finished.

**Comment by elboyran** [comment]
I will try to solve it inside the issue I'm working on #697.

**Comment by loostrum** [comment]
I see the issue, could also do a quick fix for that in a separate PR. Let me know what you prefer.
